src/super_gradients/training/models/model_factory.py

```diff
@@ -169,27 +170,29 @@ def get(
 
     NOTE: Passing pretrained_weights and checkpoint_path is ill-defined and will raise an error.
     """
-    checkpoint_num_classes = checkpoint_num_classes or num_classes
+    arch_params = arch_params if arch_params else {}
+    with warn_if_unused_params(arch_params) as arch_params:
```

Comment:
It would be great if we start reducing the amount of warning duplicates when running in DDP.
Can we do it here explicitly? or even better inside warn_if_unused_params ?

Reply:
DDP will display it only once because the loggers are muted for rank>1.
Also models.get is usually called after destup_device, so the launcher will not raise any warning.
The conclusion is that with/without DDP, it will only print once